Sources/MapboxNavigation/MapView.swift

```diff
@@ -37,6 +27,39 @@ extension MapView {
         return []
     }
     
+    /**
+     Returns a list of identifiers of the tile sets that make up specific source type.
+     
+     This array contains multiple entries for a composited source. This property is empty for non-Mapbox-hosted tile sets and sources with type other than `vector`.
+     
+     - parameter sourceIdentifier: Identifier of the source.
+     - parameter sourceType: Type of the source (e.g. `vector`).
+     - returns: List of tile set identifiers.
+     */
+    func tileSetIdentifiers(_ sourceIdentifier: String, sourceType: String) -> [String] {
+        if sourceType == "vector" {
+            return self.tileSetIdentifiers(sourceIdentifier)
+        }
+        
+        return []
+    }
+    
+    /**
+     Returns a set of source identifiers for tilesets that are or include the Mapbox Incidents source.
+     
+     - parameter tileSetIdentifier: Identifier of the tile set in the form `user.tileset`.
+     - returns: Set of source identifiers.
+     */
+    func sourceIdentifiers(_ tileSetIdentifier: String) -> Set<String> {
+        return Set(mapboxMap.style.allSourceIdentifiers.filter {
```

Comment:
Do you know how big this array grows? May be it makes sense to optimise this method a little?

Reply:
Source count is usually not really high (less than hundred I guess) so pre-mature optimization might not be beneficial.